roles/common/tasks/main.yml

```diff
@@ -18,6 +18,13 @@
   tags:
     - common
 
+# make sure yum-utils is installed
+- name: add yum-utils
+  sudo: yes
+  yum:
+    name: yum-utils
+
```

Comment:
could you add the bootstrap tag to this? We probably should install this in baked images.